src/templates/docPage.js

```diff
@@ -65,15 +65,23 @@ const BasicDoc = ({ data, location, pageContext }) => {
     title,
     metaDescription,
     type,
-    tags,
     translationType,
     dataSource,
+    isTutorial,
   } = frontmatter;
 
+  let { tags } = frontmatter;
```

Comment:
How is this component setting the metatag document type to tell DLS that this is a tutorial doc? Does it get the same info from tags?

https://github.com/newrelic/docs-website/blob/develop/src/components/SEO.js#L70

Reply:
whoops, this is just adding a tag, i'll switch it to be a type 🙊